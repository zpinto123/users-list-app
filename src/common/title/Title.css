.titleContainer {
  height: 100px;
  text-align: center;
}

.titleText {
  font-size: 2em;
  line-height: 100px;
}