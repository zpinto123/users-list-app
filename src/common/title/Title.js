import React from "react";
import PropTypes from 'prop-types';

import "./Title.css";

const Title = ({ text }) => (
  <div className="titleContainer">
    <label className="titleText">{text}</label>
  </div>
);

Title.propTypes = {
  text: PropTypes.string.isRequired
}

export default Title;
