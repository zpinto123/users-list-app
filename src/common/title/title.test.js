import React from "react";
import renderer from "react-test-renderer";

import Title from "./Title";

describe("<Title />", () => {
  it("renders correctly", () => {
    const component = renderer
      .create(<Title text="My awesome title!" />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
