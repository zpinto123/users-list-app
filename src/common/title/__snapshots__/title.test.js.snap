// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> renders correctly 1`] = `
<div
  className="titleContainer"
>
  <label
    className="titleText"
  >
    My awesome title!
  </label>
</div>
`;
